src/pint/toa.py

```diff
@@ -305,6 +308,9 @@ def get_TOAs(
     if usepickle and updatepickle:
         log.info("Pickling TOAs.")
         save_pickle(t, picklefilename=picklefilename)
+    if "pulse_number" in t.table.colnames and not include_pn:
```

Comment:
Checking `include_pn` first makes it more obvious that this is controlled by an option, and also saves doing the `in` in the common (default) case where the option isn't set.

Reply:
ok